src/rust/bitbox02-rust-c/src/workflow.rs

```diff
@@ -92,6 +110,15 @@ pub unsafe extern "C" fn rust_workflow_spin() {
             }
             _ => (),
         }
+        match ORIENTATION_SCREEN_STATE {
+            TaskState::Running(ref mut task) => {
```

Comment:
I wonder why Rust is not erroring here as it's taking a mutable ref to a static mut :thinking:

Reply:
we have `#![allow(static_mut_refs)]` at the top, and I just copy/pasted the other functions with similar code in the same file.